audio_service/darwin/Classes/AudioServicePlugin.m

```diff
@@ -206,7 +208,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
         }
         [self broadcastPlaybackState];
         [self updateControls];
-        if (playing != wasPlaying) {
+        if (playing != oldPlaying ||
+            speed.doubleValue != oldSpeed.doubleValue ||
+            position.unsignedLongValue != oldPosition.unsignedLongValue) {
```

Comment:
Although it may seem like the position will never be negative, I think this code would be more robust if we didn't make that assumption and just treated it as a long long.